### DISCUSSION TITLE
Missing refreshing for RB-SRAIN01

### DISCUSSION THREAD

**Discussion by luckbert** [Koenkk/zigbee2mqtt#23937] [Devices]
Hello, 
I have a problem with the RB-SRAIN01 rain sensor. 

I am using z2m in Homeassitant and have successfully added the sensor. Unfortunately, it does not update itself after some time. It does not change its status after an unknown time. The current status is only displayed after starting ‘interview’. 
Is this a bug in z2m or is my device defective? 
Greetings Jürgen

**Comment by luckbert** [comment]
Good morning, 
It's done. 
The error did not occur again after the last interview. 
Regards

**Reply by buhito81** [reply]
Samen situation here. What did you do to have it remain updating the status?

**Comment by luckbert** [comment]
hi, 
The first thing I did was press the interview button. 

However, I have now found out that it is due to the connection between the sensor and the base. The distance must not be too great. Max. 2-3 metres
Regards
